semgrep_output_v1.atd

```diff
@@ -1113,6 +1113,11 @@ type found_dependency = {
     <ts repr="map">;
   ?resolved_url: string option;
   transitivity: transitivity;
+  (* Path to the manifest file which the lockfile can be associated with
```

Comment:
Maybe "Path to the manifest file that defines the project containing this dependency"?